src/main/java/com/mixpanel/android/mpmetrics/GCMReceiver.java

```diff
@@ -399,6 +402,56 @@ protected Notification makeNotificationSDK21OrHigher(Context context, PendingInt
         return n;
     }
 
+    @SuppressLint("NewApi")
+    @TargetApi(26)
+    protected Notification makeNotificationSDK26OrHigher(Context context, PendingIntent intent, NotificationData notificationData) {
+        NotificationManager mNotificationManager =
+                (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+        String channelId = "mp";
```

Comment:
I think it will be better if the host app can provide the channel id, name, importance

Reply:
Thanks for your feedback!